routes/projects.rb

```diff
@@ -25,6 +25,21 @@ def self.registered(app)
           halt 404, 'Project not found'
         end
         @clients = Client.filter(id: @project.client_id).all
+        if @clients.nil?
+          MyLog.log.fatal "routes/projects: Project's clients not found"
+          halt 404, "Project's Client not found"
+        end
+        @nodes = NodeResource.filter(project_id: @project.id).all
```

Comment:
I don't think that the nodes and dbs should have an error if they aren't present, a project doesn't have to have either